frontend/src/styles/UniversalLandingPage.module.scss

```diff
@@ -0,0 +1,46 @@
+.topDesignLove {
+  justify-items: center;
+  position: absolute;
+  width: 1440;
+  height: 340;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  fill: none;
+}
+
+.nightlineDesignLeft {
+  position: absolute;
+  width: 543.62px;
+  height: 659px;
+  left: 0px;
+  top: 500px;
+}
+
+.nightlineDesignRight {
+  position: absolute;
+  width: 543.62px;
+  height: 659px;
+  left: 896.38px;
```

Comment:
Instead of setting a fixed left position like this, try putting the nightlineRight, nightlineLeft, and the button in a grid container for a more responsive design